app/herdbook.py

```diff
@@ -474,6 +478,137 @@ def get_mean_kinship(g_id):
     return {}
 
 
+@APP.route("/api/certificates/issue/<i_number>")
+@login_required
+def issue_certificate(i_number):
+    """
+    Returns an issued pdf certificate of the individual given by `i_number`.
+    """
+    user_id = session.get("user_id", None)
+    ind = da.get_individual(i_number, user_id)
+
+    if ind:
+        all_data = get_all_data(ind, user_id)
+        pdf_bytes = get_certificate(all_data)
+        date = datetime.datetime.utcnow()
+        date = date.strftime("%Y-%m-%d_%H%M%S%f")
+        object_name = f"{date}.pdf"
```

Comment:
Can we generate a more usable file name?

Reply:
I guess so! We need to decide on a pattern for that. I am open tu suggestions.

Reply:
The number of the rabbit is welknowed I suggest number-certificate.pdf

So like G1274-2001-5678966.pdf

Maybe date but that's probably to much